src/main/java/hu/eszter/bokkon/service/Initializer.java

```diff
@@ -17,14 +15,26 @@ public static AnimalBaseStock createAnimalStock() {
 
     public static Die createDice(Animal a1, Animal a2, Animal a3) {
         List<Animal> diceSides = new ArrayList<>();
-        diceSides.addAll(Stream.generate(Rabbit::new).limit(6).collect(Collectors.toList()));
-        diceSides.addAll(Stream.generate(Sheep::new).limit(2).collect(Collectors.toList()));
-        diceSides.add(new Pig());
+        for (int i = 0; i < 6; i++) {
+            diceSides.add(Animal.RABBIT);
+            if (i < 2) {
+                diceSides.add(Animal.SHEEP);
+            }
+        }
+        diceSides.add(Animal.PIG);
         diceSides.addAll(Arrays.asList(a1, a2, a3));
         return new Die(diceSides);
     }
 
     public static Farmer createPlayer(String name) {
         return new Farmer(name);
     }
+
+    public static ExchangeService setUpExchangeService() {
+        return new ExchangeService();
+    }
+
+    public static DiceRollService setUpDiceRollService() {
+        return new DiceRollService();
```

Comment:
I would prefer passing two dice to the service's constructor, instead of DiceRollService calling back to the Initializer to create them